lib/heartcheck/controllers/environment.rb

```diff
@@ -0,0 +1,17 @@
+require 'sys-uname'
+
+module Heartcheck
+  module Controllers
+    class Environment < Base
+      def index
+        MultiJson.dump(
+          {
+            'system-info' => Sys::Uname.uname,
+            'ruby-version' => RUBY_VERSION,
+            'rails-version' => defined?(Rails) ? Rails::VERSION::STRING : '(none)'
+          }
+        )
+      end
+    end
+  end
+end
```

Comment:
I think it is missing a test for this controller

Reply:
Yes, I just pushed the tests